sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -178,10 +187,13 @@ The number of cycles between when the ready signal is deasserted and when the
 pipe can no longer accept new inputs.
 
 This property is not guaranteed to be respected if the pipe is an inter-kernel
-pipe. The compiler is allowed to optimize the pipe if both sides are visible.
+pipe. 
+
+This property will is only valid when the `protocol` property is *avalon_streaming*
+or *avalon_mm*.
```

Comment:
I don't think it should be allowed with avalon_mm.  First of all, readyLatency is only a property of avalon streaming interfaces but secondly, even if it did make sense on avalon-mm interfaces, in this case avalon-mm means that the pipe will be accessed through a single shared avalon-mm interface with possibly other pipes, kernel arguments, device_globals, and kernel controls.  It doesn't make sense for each of those entities to individually control the properties of the interface as these can't contradict each other.  It would make more sense to control these properties (if we believe we need to provide controls for them) through compiler flags.